frontend/src/routes/[[lang=locale]]/candidate/(protected)/preview/+page.svelte

```diff
@@ -12,23 +12,33 @@
   const {userStore} = getContext<CandidateContext>('candidate');
 
   let infoQuestions: QuestionProps[];
-  getInfoQuestions({locale: $locale}).then((res) => (infoQuestions = res));
-
   let opinionQuestions: QuestionProps[];
-  getOpinionQuestions({locale: $locale}).then((res) => (opinionQuestions = res));
-
   let candidate: CandidateProps | undefined;
-  const loadCandidate = async () => {
-    const res = await getNominatedCandidates({
-      loadAnswers: true,
-      locale: $locale,
-      id: $userStore?.candidate?.id.toString()
-    });
-    candidate = res[0];
+  let loadData: Promise<void>;
+
+  const fetchData = async () => {
+    const [infoRes, opinionRes, candidateRes] = await Promise.all([
+      getInfoQuestions({locale: $locale}),
+      getOpinionQuestions({locale: $locale}),
+      getNominatedCandidates({
+        loadAnswers: true,
+        locale: $locale,
+        id: $userStore?.candidate?.id.toString()
+      })
+    ]);
+
+    infoQuestions = infoRes;
+    opinionQuestions = opinionRes;
+    candidate = candidateRes[0];
   };
+
+  $: {
+    loadData = fetchData();
+    $locale;
+  }
```

Comment:
The `fetchData` function doesn't return anything but the return value is assigned to `loadData` on line 36. What is happening here? :D